guide/daily-spending-wallet/send-fees.md

```diff
@@ -0,0 +1,205 @@
+
+{% include image-gallery.html pages = page.images_breakdown %}
+
+#### Routing fees
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lightning-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lightning-fee-sample@2x.png"
+   alt-text = "How to display a lightning routing fee"
+   width = 400
+   height = 85
+   layout = "float-right-desktop"
+%}
+
+On the lightning network, payments are passed between nodes to get from the sender to the receiver. Each of those nodes may charge a base fee and a second fee based on a percentage of the amount forwarded. Fees paid can vary, but are typically in the single-digit or double-digit Satoshi range (a small fraction of on-chain fees).
+
+</div>
+
+#### Service fees
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lsp-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/lsp-fee-sample@2x.png"
+   alt-text = "How to display a lightning service fee"
+   width = 400
+   height = 85
+   layout = "float-right-desktop"
+%}
+
+Lightning wallets may require [additional services]({{ '/guide/how-it-works/lightning-services/' | relative_url }}) to solve certain usability issues. An example being a lightning service provider (LSP) opening payment channels and providing [inbound liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}) for the user, so they can receive payments. As these services are offered by third parties, additional fees may be charged.
+
+</div>
+
+## On-chain fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample.png"
+   retina = "/assets/images/guide/daily-spending-wallet/sending/send-fees/onchain-fee-sample@2x.png"
+   alt-text = "How to display an onchain transaction fee"
+   width = 400
+   height = 252
+   layout = "float-right-desktop"
+%}
+
+This fee is largely dependent on how many other transactions are currently [waiting]({{ '/guide/glossary/#mempool' | relative_url }}) to be processed on the base layer as a whole. The [average fee](https://ycharts.com/indicators/bitcoin_average_transaction_fee) in January 2021 was $0.63, and $28.60 in April 2021.
+
+A new block of transactions is confirmed roughly every 10 minutes. Blocks have size limits, which places a limit on the number of transactions they can contain. Transactions size itself varies with complexity. A complex multi-key transaction with many inputs and outputs involves more data, and therefore carries a higher fee, than a simple one that involves only one input and output. An organic fee market emerges from these dynamics, as miners try to earn as much as they can from fees, and transacting users try to optimize for time, fee rate, or other priorities.
```

Comment:
```suggestion
A new block of transactions is confirmed roughly every 10 minutes. Blocks have a size limit, which places a limit on the number of transactions they can contain. Transactions size itself varies with complexity. A complex multi-key transaction with many inputs and outputs involves more data, and therefore carries a higher fee, than a simple one that involves only one input and output. An organic fee market emerges from these dynamics, as miners try to earn as much as they can from fees, and transacting users try to optimize for time, fee rate, or other priorities.
```

I agree with Mo's suggestions. Also, I don't think we need plural "size limits". There are not _multiple size limits_. Certainly, the size limit is a little complicated to think about since the witness is discounted, but the size limit is still a singular 4MB.

Reply:
Thanks. I manually applied this change since Mo had already changed this line and GitHub wouldn't allow me to apply both suggestions at the same time.